pandoc/default_lab_docs.yaml

```diff
@@ -0,0 +1,15 @@
+from: rst
```

Comment:
This file is "yaml" but ".github/workflows/main.yml" is "yml" - consistency would be helpful (if we can)

Reply:
Yes. It seems to be preferred convention to name them `.yaml` indeed

Reply:
Oh I misunderstood. I will rename .github/workflows/main.yml -> yaml, it is supposed to work